robot/session_test.go

```diff
@@ -786,16 +796,17 @@ func TestSessionsMixedOwnersImplicitAuth(t *testing.T) {
 	test.That(t, resp.Id, test.ShouldNotEqual, sessID)
 	test.That(t, resp.Id, test.ShouldNotEqual, "")
 
-	// this is the only one heartbeating so we expect a stop
+	// Assert that closing `roboClient1` results in heartbeats stopping that propagates to `Stop`ing
```

Comment:
roboClient1 -> robotClient1

Reply:
The variable name here is `roboClient1`. Though I agree the variable name of `robotClient1` would have been a bit more conventional.